naucse/templates/lesson.html

```diff
@@ -25,6 +25,15 @@
                 <hr>
             </header>
         {% endif %}
+        
+        {% if course is defined and not course.canonical %}
+            <div class="alert alert-info">
+                Toto je stránka lekce z kurzu, který probíhá nebo proběhl naživo s instruktorem.
+                {% if course.derives %}
+                    <a href="{{ canonical_url }}">Přejít na stejnou lekci v kurzu pro samouky</a>.
+                {% endif %}
+            </div>
+        {% endif %}
```

Comment:
I don't find it too obtrusive, but it doesn't have to be presented like this, I just believe the link should be there somewhere.  It could be at the bottom of the page in smaller text.